src/pages/admin/ApplicationPage.vue

```diff
@@ -9,6 +9,7 @@
         <DataTable />
       </q-card-section>
     </q-card>
+    <router-view />
```

Comment:
router view is for?

Reply:
that is temporary, to show that the page for the children of the parent route is working.